airsonic-main/src/main/java/org/airsonic/player/dao/UserDao.java

```diff
@@ -224,7 +224,7 @@ public void updateUserSettings(UserSettings settings) {
                 settings.getAvatarScheme().name(), settings.getSystemAvatarId(), settings.getChanged(),
                 settings.isShowArtistInfoEnabled(), settings.isAutoHidePlayQueue(),
                 settings.isViewAsList(), settings.getDefaultAlbumList().getId(), settings.isQueueFollowingSongs(),
-                settings.isShowSideBar(), settings.getListReloadDelay(), settings.isKeyboardShortcutsEnabled(),
+                settings.isShowSideBar(), 60 /* Unused listReloadDelay */, settings.isKeyboardShortcutsEnabled(),
```

Comment:
Can we remove this column instead?

Reply:
As I noted in the original PR comment, I chose not to remove the column and just code around it so that it would not cause a problem if users downgraded or otherwise switched between versions.  I think it would be appropriate to remove it when we are preparing for the next major release, but for now I think it's safer not to.

Reply:
Can you maybe add a todo for us? Maybe a github issue so we can track it?